docs/docs/guides/tools/comparison.md

```diff
@@ -0,0 +1,113 @@
+# Comparison
+
+The Weave Comparison feature allows you to visually compare and diff code, traces, prompts, models, model configs, evaluation metrics, and evaluation predictions.  You can compare two objects side-by-side or analyze a larger set of objects to identify differences, patterns, and trends.
+
+This guide covers the steps to start a comparison and the available actions to tailor your comparison view, including baseline comparisons, numeric diff formatting, and more. 
+
+> Make sure to look at the [Usage notes](#usage-notes).
+
+## Access the Comparison view
+
+1. In the sidebar, select the type of object you'd like to compare (e.g. **Traces**, **Models**, etc.).
```

Comment:
There's a missing step here for objects, where you have to go to the versions page for each object to get the checkboxes.

Technically, the comparison page would work if you specified different objects in the URL, but since that is only useful in rare cases we haven't wired it up to any user actions in the UI.

Reply:
Updated the step to account for Models in https://github.com/wandb/weave/pull/3124/commits/65d43eb765868268559f8573347bdcc4a3302dcd